### ISSUE TITLE
understanding versioning scheme of charts

### ISSUE THREAD

**Issue by KlavsKlavsen** [codecentric/helm-charts#810]
I see you updated the keycloakx chart from 3.0.0 to 4.0.0 on the 20th of december.. and then from 4.0.0 to 5.0.0 on 27th of december. Only change is to the chart version - no change in AppVersion.. so as far as I can see this change brings zero changes to the actual yaml generated from the chart.

What am I missing? whats the reason behind the chart major version upgrades?

**Comment by martinmeneval** [comment]
It almost seems as though it's an automatic version bump, on a bot that has run amok? I don't believe the committer @mircohaug is an actual user, or perhaps it is @mircohacker who changed names?

**Comment by lusu007** [comment]
It appears that every chart receives a version bump whenever any chart is updated. For instance, in Mailhog Chart version 5.7.0, there were no changes for Mailhog itself, but the KeycloakX AppVersion was updated.

**Comment by bguijt** [comment]
Exactly. The commit history is just silly major version bumps: https://github.com/codecentric/helm-charts/commits/master/charts/keycloakx

**Comment by KlavsKlavsen** [comment]
@jonesbusy @megian any of you know who manages the bot thats updating versions in this chart all the time with no good reason?

**Comment by jonesbusy** [comment]
No idea, I'm not maintainer of those charts

**Comment by megian** [comment]
I'm also not a maintainer.

**Comment by KlavsKlavsen** [comment]
@pascal-sochacki seems to be the one who has privs to be able to stop this bot from updating the chart major version needlessly every week :)

**Comment by pascal-sochacki** [comment]
I disabled the workflow for now. I inherited the repository because no one was maintaining it. I also need to figure out how the release workflow works and update to newer versions. I am aware I'm doing a bad job maintaining it.

**Comment by KlavsKlavsen** [comment]
@pascal-sochacki no worries man. Glad to hear the forced major version upgrades, with no changes have been disabled now :)

If you share a plan for the future somewhere  - then maybe others can see if there's some of those they want to pickup? You've already gotten PRs merged previously from others - so you seem to be doing a fine enough job.

We use Keycloakx chart - so we'll happily assist if needed.

**Comment by mircohacker** [comment]
Yes I wrote this automation. It analyses the commit messages in order to find out, what kind of release should happen. It seems like it does not find the correct commit any more. But as @pascal-sochacki indicated I am no longer in a position to fix this personally. You can starte [here](https://github.com/codecentric/helm-charts/tree/master/.github/actions/bumpVersionAction) to find out what broke exactly.
